Makefile

```diff
@@ -13,7 +11,8 @@ clean:
 build-full: clean build
 
 build-ui:
-	cd ui && npm run build
+	npm install webpack-dev-server rimraf webpack react-scripts -g
```

Comment:
is this Github Action specific? or its required by the react build process? If required by build process, should we just add it into frontend `package.json`